### DISCUSSION TITLE
2023년 03월 09일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#5] [General]
# 2023년 03월 09일

LOG 블랙캣과 점심을 먹었다. 역시나 신천 직화ㅋㅋ 페어 미션 재밌다. 재밌는 주제 생각해준 이리내 👍 오늘 아침에도 소파 옆쪽 자리(?)가 있어서 한 번 앉아봤다. 확실히 굿샷보다는 낫긴 한데 포이가 앉은 것처럼 의자에 앉는게 훨씬 좋은 것 같기…

https://cyma-s.github.io/2023-03-08/

**Comment by carsago** [comment]
잘 보고 갑니다~

**Reply by Cyma-s** [reply]
많은 관심 감사합니다~~ ^~^ 👍

**Comment by ezzanzzan** [comment]
로베로베 많관부 %^^..  총총
